rerun_cpp/src/rerun/archetypes/bar_chart_ext.cpp

```diff
@@ -1,12 +1,9 @@
 #include "../half.hpp"
 #include "bar_chart.hpp"
 
-// #define EDIT_EXTENSION
+namespace rerun::archetypes {
 
-namespace rerun {
-    namespace archetypes {
-
-#ifdef EDIT_EXTENSION
+#if 0
```

Comment:
`#if 0` looks wrong, though I know it is right. Perhaps add a comment, or standardize how we do this

Reply:
Hmmm. Need to do a pass over these. Slowly switched to the simpler if 0 variant because the other thing got implemented incorrectly too often